containers/Makefile

```diff
@@ -16,9 +16,9 @@ ifndef CONCENT_DIR
 endif
 $(info CONCENT_DIR=$(CONCENT_DIR))
 
-all: build/ nginx-proxy concent-api
+all: build/ concent-api nginx-proxy
 
-nginx-proxy: $(NGINX_PROXY_SOURCE)
+nginx-proxy: $(NGINX_PROXY_SOURCE) concent-api build/nginx-proxy/concent-api-assets.tar
```

Comment:
There's no reason for `nginx-proxy` docker image to depend on the `concent-api` image. One does not use the other.